spin/spin.go

```diff
@@ -73,7 +74,13 @@ func (m model) Init() tea.Cmd {
 		commandStart(m.command),
 	)
 }
-func (m model) View() string { return m.spinner.View() + " " + m.title }
+func (m model) View() string {
+	if (m.location == "left") {
```

Comment:
This might seem odd coming from C, but in Go you tend to avoid the parentheses if there's only a single conditional. I'll push a fix to your fork.

Reply:
Pro tip: in the Go world there's a standard tool called `gofmt` that automatically formats your sources according to the language conventions. It even fixes little Go characteristics like the missing parentheses.

More about it here: https://go.dev/blog/gofmt

I realize you're just dabbling in our Go code to post this quick fix, but I figured it can't harm to point this out :smile:

Reply:
> This might seem odd coming from C, but in Go you tend to avoid the parentheses if there's only a single conditional. I'll push a fix to your fork.

I much prefer this style (it's the kernel style to not have a { } for only a single conditional), I wasn't sure about the Go style, sorry.